fluent-bit-cloudwatch.go

```diff
@@ -141,6 +141,11 @@ func getBoolParam(ctx unsafe.Pointer, param string, defaultVal bool) bool {
 func FLBPluginInit(ctx unsafe.Pointer) int {
 	plugins.SetupLogger()
 
+	logrus.Debug("A new higher performance CloudWatch Logs plugin has been released;" +
```

Comment:
I want this message to be in the code... as a way to increase the chances that people realize they are using an old plugin. But I also want it to be a debug statement, so that it is not annoying- since this plugin has unique features that some folks will continue to need.